### ISSUE TITLE
ember install:addon blueprint arguments

### ISSUE THREAD

**Issue by jakecraige** [ember-cli/ember-cli#2846]
### Overview

I've noticed a problem that I'm not sure exactly how we should fix so I'd like some feedback. I'm happy to implement the fix once we have a clear path.

Currently the `install:addon` command will [install multiple packages](https://github.com/ember-cli/ember-cli/blob/master/lib/commands/install-addon.js#L33) but only [runs the blueprint for the first one](https://github.com/ember-cli/ember-cli/blob/master/lib/commands/install-addon.js#L54). That is a fixable problem, but it opens up a new one.

For ember-cli-cordova. I want to run `ember install:addon ember-cli-cordova com.myapp.app`. and have it run my generator with the the extra `rawArg`. In it's current state, it tries to also `npm install com.myapp.app` which doesn't exist. 

### tldr

The current install addon only runs the generator for the *first* addon given and rawArgs won't flow through since it will try to npm install the args.

###  Possible solutions:

1. Only allow a single addon to be installed at a time. This would allow addons to receive extra args properly.
2. Don't support addons requiring args for their generators with the `install:addon` system.
3. Implement some sort of `--addon-args` option that will be turned into args that get passed to the addons. The problem with that is that it would have to support multiple addons being installed, so either we'd have to pass it a big string somehow prefixed with the addon name or the order of your args could be made to matter.

### What I'm thinking

Even though it's more work for my project, I'm thinking 2 would be the best approach and make the most sense for people. People who needs their generators to take options can just have users run the generator manually with options

**Comment by jakecraige** [comment]
Sounds fine with me. #2919 closes this by slicing off the first name as the package, and the rest become the extra args passed to the blueprint
